### PR TITLE
Updated : Issue 1978 - Changes to footer based on `intranet` implemen…

### PR DESCRIPTION
**Pull request by bhushan-ebi**
…tation findings

### CODE REVIEW

File: components/vf-footer/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+### 1.2.1
+
+* Changes to footer based on `intranet` implementation findings [Tracking issue](https://github.com/visual-framework/vf-core/issues/1978)
```
Comment (sandykadam) [inline components/vf-footer/CHANGELOG.md]: rename it to "EMBL-EBI Intranet"

**Comment by sandykadam** [comment]
@bhushan-ebi are we supposed to make changes in ["ebi-header-footer"](https://stable.visual-framework.dev/components/ebi-header-footer/index.html?q=footer) component or generic component of "vf-footer" ?

File: components/vf-footer/CHANGELOG.md (hunk 1 above)
Comment (bhushan-ebi) [reply components/vf-footer/CHANGELOG.md]: @sandykadam change done - https://github.com/visual-framework/vf-core/pull/2113/commits/a7b9c5286943ed09f0a25a7cfa7b6bfe6264fdba

**Comment by bhushan-ebi** [comment]
> @bhushan-ebi are we supposed to make changes in ["ebi-header-footer"](https://stable.visual-framework.dev/components/ebi-header-footer/index.html?q=footer) component or generic component of "vf-footer" ?

@sandykadam changes are recommended for generic 'vf-footer'

**Review by sandykadam — APPROVED** [review]

**Pull request merged**